host_applications/linux/apps/gencmd/gencmd.c

```diff
@@ -130,6 +153,14 @@ int main( int argc, char **argv )
           {
               printf("%s\n", buffer );
           }
+          if (strncmp( buffer, "error=", 5) == 0 )
+          {
+             if ( strcmp( buffer, "error=1 error_msg=\"Command not registered\"" ) == 0 )
+             {
+                printf( "Use 'vcgencmd commands' to get a list of commands\n" );
```

Comment:
Should this be printing to STDERR?

Reply:
Good point. None of the other errors are getting printed to STDERR so I think I will leave that to a future PR.

Reply:
I've now got all errors going to stderr